theme/static/css/hs-file-icons.css

```diff
@@ -98,7 +98,9 @@
 }
 
 .fb-view-list .fb-stack .fa-file-o {
-	font-size: 18px;
+    font-size: 18px;
```

Comment:
@engineSound - could you review the frontend html structures and styling?

Reply:
@sblack-usu The html and css overall looks good to me. Just some minor comments here and there. Maurier has a good work on the html and css. On the same line, we need to discuss separately where and how we encapsulate html snippets to avoid them to spread everywhere #3097.